portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/permissions/Permissions.testcase

```diff
@@ -83,7 +83,7 @@ definition {
 	@description = "This is a use case for LRQA-42712. Verify guest users cannot configure public pages."
 	@priority = "4"
 	@uitest
-	test ViewNoDisplayPortletConfigurationPermission {
+	test ViewNoDisplayPortletConfigurationPermissionAsGuest {
```

Comment:
The other test case which you rewrote is similiar in description and in goal too. For these reasons and in the aspect of consistency I would suggest using similiar naming like on the other case (It was "**CannotConfigureBlogsPortletAsGuestPG**")

Reply:
I agree

Reply:
Done!